prow/cmd/prow-controller-manager/README.md

```diff
@@ -0,0 +1,23 @@
+# Prow-Controller-Manager
+
+`prow-controller-manager` manages the job execution and lifecycle for jobs running in k8s. It is a revision of [Plank](/prow/cmd/plank).
```

Comment:
Its actually intended to eventually contain all controllers: https://github.com/kubernetes/test-infra/issues/17024
Plankv2 is so far the only one that got integrated, but I hope to continue the work on that soon

Reply:
Fixed, and reworded the doc a bit to show it's a work in progress but also works (for Plank)